### PR TITLE
Add support for BalancerV2ComposableStablePoolFactoryV6 in contracts

### PR DESCRIPTION
**Pull request by mateo-mro**
# Description
Add support for BalancerV2ComposableStablePoolFactoryV6 in contracts. It is compatible with V5, so no changes are needed: https://github.com/balancer/balancer-deployments/pull/108/files#diff-8730c2b8ae6de974591d1c41121ef43dc296a98f578c960359f328e9d54afe50R3-R4

### CODE REVIEW

File: crates/contracts/build.rs [hunk 1]
```diff
@@ -534,6 +534,51 @@ fn main() {
...
+            .add_network(
+                ARBITRUM_ONE,
+                Network {
+                    address: addr("0x4bdCc2fb18AEb9e2d281b0278D946445070EAda7"),
+                    // <https://arbiscan.io/tx/0xfa1e7642e135fb32dc14c990b851e5e7a0ac7a463e3a60c5003ae4142396f45e>
+                    deployment_information: Some(DeploymentInformation::BlockNumber(184805448)),
+                },
+            )
+            .add_network(
+                ARBITRUM_ONE,
```
Comment (MartinquaXD) [inline crates/contracts/build.rs:574]:
```suggestion
                Base,
```

File: crates/contracts/artifacts/BalancerV2ComposableStablePoolFactoryV6.json
Comment (MartinquaXD) [inline crates/contracts/artifacts/BalancerV2ComposableStablePoolFactoryV6.json:1]: Shows up weird in the github UI but when checked out the file content looks correct (identical to v5).

**Review by MartinquaXD — APPROVED** [review]
Approved assuming the second arbitrum entry gets fixed.

**Review by sunce86 — APPROVED** [review]
LGTM

**Review by squadgazzz — APPROVED** [review]

**Pull request merged**